README.md

```diff
@@ -20,9 +20,105 @@
 1. To setup the app you want to first `fork` repository from [GitHub](https://github.com/Creams-Quad/bounty-back)
 2. In the fork repository click the green `code` tab. A drop down to copy the repository link or url will be shown. Copy that link.
 3. Go to your local machine terminal and `cd` into a directory or folder to copy link.
-4. In that directory or folder type in `git clone (with the copylink)`
+4. In that directory or folder type in `git clone (with the copied link)`
 5. Type in `npm install` to install all dependencies for the app.
 
+## Routes
+
+### /bounties:
+  - POST
+    - description: create a new bounty with an id and add it to the bounties table
+    - status code:(200)
+    - JSON input:
+      - username:{string}
+      - password:{string}
+  - GET
+    - description: 
+    - status code:(200)
+    - no JSON Input
+
+### /bounties/id:
+  - PUT
+    - description: update a bounty with in the bounties table on a given id
+    - status code:(200)
+    - JSON input:
+      - header:{string}
+      - content:{string}
+      - poster:{string}  
+      <!-- - username:{string}
+      - password:{string} -->
+  - GET
+    - description: return a bounty with related comments
+    - status code:(200)
+    - no JSON input
+    - Example output
+       ```JSON
+      {
+      "id":1,
+      "header":"foo",
+      "content":"bar",
+      "poster":"John Doe",
+      "karma":100,
+      "comments": [
+        {
+          "id":1,     
+          "bountyId":1,
+          "header":"fizz",
+          "content":"buzz",
+          "poster":"Holly Doe",
+          "karma":100
+        }]
+      }
+  - DELETE
+    - description: delete bounty from table
+    - status code:(204)
+    - no JSON input
+  
+
+### /comments:
+  - POST
+    - description: add a comment to comments table
```

Comment:
I know this is really a semantic issue... but can you change "add" to "creates". The sentence should read, "Creates a comment record"...